frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/contextmenu/ShowSelectedInGitLogAction.java

```diff
@@ -49,36 +49,38 @@ protected void onUpdate(AnActionEvent anActionEvent) {
     val selectedBranchName = getSelectedBranchName(anActionEvent);
     // It's very unlikely that selectedBranchName is empty at this point since it's assigned directly before invoking this
     // action in GitMacheteGraphTable.GitMacheteGraphTableMouseAdapter.mouseClicked; still, it's better to be safe.
-    if (selectedBranchName.isEmpty()) {
+    if (selectedBranchName == null || selectedBranchName.isEmpty()) {
       presentation.setEnabled(false);
       presentation.setDescription(getNonHtmlString("action.GitMachete.ShowSelectedInGitLogAction.undefined.branch-name"));
       return;
     }
 
     presentation.setDescription(
         getNonHtmlString("action.GitMachete.ShowSelectedInGitLogAction.description.precise")
-            .format(selectedBranchName.get()));
+            .format(selectedBranchName));
   }
 
   @Override
   @UIEffect
   public void actionPerformed(AnActionEvent anActionEvent) {
     val selectedBranchName = getSelectedBranchName(anActionEvent);
-    if (selectedBranchName.isEmpty()) {
+    if (selectedBranchName == null || selectedBranchName.isEmpty()) {
       return;
     }
 
     val project = getProject(anActionEvent);
     val gitRepository = getSelectedGitRepository(anActionEvent);
 
-    if (gitRepository.isDefined()) {
-      log().debug(() -> "Queuing show '${selectedBranchName.get()}' branch in Git log background task");
+    if (gitRepository != null) {
+      log().debug(() -> "Queuing show '${selectedBranchName}' branch in Git log background task");
 
-      GitBranchesCollection branches = gitRepository.get().getBranches();
-      @SuppressWarnings("nullness:return") val maybeHash = selectedBranchName
+      val branches = gitRepository.getBranches();
+
+      @SuppressWarnings("nullness:return") val maybeHash = Option.of(selectedBranchName)
```

Comment:
Huh Checker doesn't support generics 100% correctly (yet), some false positives still do happen (as I suppose is the case here)